vision/unit_tests/test_client.py

```diff
@@ -15,16 +15,16 @@
 
 import unittest
 
-from google.cloud._helpers import _to_bytes
-
-_IMAGE_CONTENT = _to_bytes('/9j/4QNURXhpZgAASUkq')
-_IMAGE_SOURCE = 'gs://some/image.jpg'
-
 
 class TestClient(unittest.TestCase):
     import base64
+    from google.cloud._helpers import _to_bytes
+    from google.cloud._helpers import _bytes_to_unicode
```

Comment:
Since we are using them to define constants (which could also be at module scope, and reduce duplication), I would say module scope.